README.md

```
@@ -1,238 +1,115 @@
-### Steps to build the Documentation locally
-1. Install UV: Follow the official instructions [here](https://docs.astral.sh/uv/#installation).
-2. Clone this repository, use command `git clone https://github.com/exospherehost/exospherehost.git`
-3. Install dependencies by navigating to `docs` folder and executing `uv sync`
-4. Use the command `uv run mkdocs serve` while `docs` folder being your working path.
+For comprehensive documentation and guides, check out:
 
-### Contribute to Documentation
-We encourage contributions to the documentation page, you can simply add a new PR with the `documentation` label.
 
-## Open Source Commitment
+- **[Getting Started Guide](https://docs.exosphere.host/exosphere/getting-started/) 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)